requirements.txt

```diff
@@ -1,5 +1,29 @@
-tornado
-futures
-python-memcached
-psutil
-docker-py
+tornado==4.4.1 \
+    --hash=sha256:371d0cf3d56c47accc66116a77ad558d76eebaa8458a6b677af71ca606522146
+futures==3.0.5 \
+    --hash=sha256:f7f16b6bf9653a918a03f1f2c2d62aac0cd64b1bc088e93ea279517f6b61120b
+python-memcached==1.58 \
+    --hash=sha256:2775829cb54b9e4c5b3bbd8028680f0c0ab695db154b9c46f0f074ff97540eb6
+psutil==4.3.1 \
+    --hash=sha256:38f74182fb9e15cafd0cdf0821098a95cc17301807aed25634a18b66537ba51b
+docker-py==1.10.3 \
+    --hash=sha256:96979a5139ce0e41ef331705b120508ce602e68097e5a21d950c14a66eb8c643
+
+backports.ssl_match_hostname==3.5.0.1 \
+    --hash=sha256:502ad98707319f4a51fa2ca1c677bd659008d27ded9f6380c79e8932e38dcdf2
+singledispatch==3.4.0.3 \
+    --hash=sha256:833b46966687b3de7f438c761ac475213e53b306740f1abfaa86e1d1aae56aa8
+certifi==2016.9.26 \
+    --hash=sha256:a60073d7bb5cb01671877efda875e8dab755b98baedfbb9b8b3d9be2d52ad022
+backports_abc==0.4 \
+    --hash=sha256:c64508e766dfe09a94a442c12b57c6e098a402921ecb340a4ec57c7e10fd464c
+six==1.10.0 \
+    --hash=sha256:0ff78c403d9bccf5a425a6d31a12aa6b47f1c21ca4dc2573a7e2f32a97335eb1
+requests==2.10.0 \
+    --hash=sha256:09bc1b5f3a56cd8c48d433213a8cba51a67d12936568f73b5f1793fcb0c0979e
+ipaddress==1.0.17 \
+    --hash=sha256:a42764805223838ec9d04eee7fe699083bf5321001792b6076322784c5b994a5
+websocket-client==0.37.0 \
+    --hash=sha256:678b246d816b94018af5297e72915160e2feb042e0cde1a9397f502ac3a52f41
+docker-pycreds==0.2.1 \
+    --hash=sha256:58d2688f92de5d6f1a6ac4fe25da461232f0e0a4c1212b93b256b046b2d714a9
```

Comment:
@peterbe wrote hashin which makes these a lot easier to maintain:

https://pypi.python.org/pypi/hashin

Figured I'd mention it to ease future maintenance burden.